document-models/arb-ltip-grantee/schema.graphql

```diff
@@ -0,0 +1,195 @@
+scalar DateTime
+
+type ArbLtipGranteeState {
+	id: ID!
+	granteeName: String
+	grantSize: Float
+	matchingGrantSize: Float
+	grantSummary: String
+	fundingAddress: String
+	fundingType: [FundingType]
+	disbursementContractAddress: String
+	metricsDashboardLink: String
+	phases: [Phase]
+}
+
+enum FundingType {
+	EOA
+	Multisig
+	TwoofThreeMultisig
+	ThreeofFiveMultisig
+}
+
+type Phase {
+	startDate: DateTime!
+	endDate: DateTime!
+	actuals: GranteeActuals
+	planned: GranteePlanned
+	stats: GranteeStats
+	status: Status
+}
+
+enum Status {
+	Uninitialized
+	NotStarted
+	InProgress
+	Finalized
+}
+
+
+type GranteeActuals {
+	arbReceived: Float
+	arbUtilized: Float
+	arbRemaining: Float
+	contractsIncentivized: [Contract]
+	summary: String
+	disclosures: String
+}
+
+type GranteePlanned {
+	arbToBeDistributed: Float
+	contractsIncentivized: [Contract]
+	distributionMechanism: [DistributionMechanism]
+	summary: String
+	summaryOfChanges: String
+}
+
+type GranteeStats {
+	avgDailyTVL: Float
+	avgDailyTXNS: Float
+	avgDailyVolume: Float
+	uniqueAddressesCount: Float
+	transactionFees: Float
+}
+
+type Contract {
+	contractId: ID!
+	contractLabel: String
+	contractAddress: String
+}
+
+enum DistributionMechanism {
+	LPIncentives
+	Airdrop
+}
+
+
+
+
+# General
+
+input SetGranteeInput {
+    id: ID!
+}
+
+
+input SetGranteeNameInput {
+	granteeName: String!
+}
+
+
+input SetGranteeGrantSizeInput {
+	grantSize: Float!
+	matchingGrantSize: Float!
+}
+
+
+input SetGranteeSummaryInput {
+	grantSummary: String!
+	fundingAddress: String!
+	fundingType: [FundingType!]!
+	disbursementContractAddress: String!
+}
+
+enum FundingType {
+	EOA
+	Multisig
+	TwoofThreeMultisig
+	ThreeofFiveMultisig
+}
+
+
+input SetGranteeMetricsDashInput {
+    metricsDashboardLink: String!
+}
+
+# add new types here
+
+
+# Metrics
+
+input InitPhaseInput {
+    startDate: DateTime!
+	phaseDuration: Int 
+    numberOfPhases: Int
+}
+
+scalar DateTime
```

Comment:
I see `scalar DateTime` is declared several times in this file — not sure if that's just an artifact of the codegen, but I'd recommend doing it once at the top.